docs/pesin_satis.md

```diff
@@ -27,8 +27,8 @@ Aşağıdaki kod örneği ödeme isteğini *Client* nesnesi üzerindeki kurucu m
 $request = $client->createPaymentRequestBuilder()
     ->buildCard('4282209027132016', '123', 5, 2019)
     ->buildOrder('ORDERNO123456', '127.0.0.1')
-    ->buildBilling('John', 'Smith', 'test@test.net', '05321231212')
-    ->buildAndAddProduct('The Product', 'PR1', 1, 10)
+    ->buildBilling('John', 'Smith', 'test@test.net', '05321231212', null, null, null, null, null, 'TR')
+    ->buildAndAddProduct('The Product', 'PR1', 1, 10, 5)
```

Comment:
@ibrahimgunduz34 Buradaki son parametre price değeri (bu örnekte 5) ve price değerini PayU tarafına gönderince aşağıdaki raw response'ı dönüyor.

```xml
<?xml version="1.0"?>
<EPAYMENT>
  <REFNO></REFNO>
  <ALIAS></ALIAS>
  <STATUS>INPUT_ERROR</STATUS>
  <RETURN_CODE>DIFFERENT_PRODUCT_ARRAY_COUNT</RETURN_CODE>
  <RETURN_MESSAGE>The product arrays have different sizes (ORDER_PCODE, ORDER_PRICE, ORDER_QTY)!</RETURN_MESSAGE>
  <DATE>2015-08-20 17:57:55</DATE>
  <ORDER_REF>ORDERNO123456</ORDER_REF>
  <AUTH_CODE></AUTH_CODE>
  <RRN></RRN>
  <HASH></HASH>
</EPAYMENT>
```

Yukarıdaki örnekte eğer price girilmez ise peyi tarafına `PaymentRequestSerializer:serializeBasket`da null olarak atanıyor ve PayU'ya gönderiliyor. 

Bunun için PayU tarafında bir dökümana baktım ancak bu hata ile ilgili bir şey bulamadım.